components/profile/MessagesSection.js

```diff
@@ -0,0 +1,72 @@
+/* globals promptError: false */
+import { useEffect, useState } from 'react'
+import api from '../../lib/api-client'
+import Table from '../Table'
+
+const MessagesSection = ({ email, accessToken }) => {
+  const [messages, setMessages] = useState([])
+  const [isLoading, setIsLoading] = useState(true)
+
+  const loadMessages = async () => {
+    try {
```

Comment:
It would be good to setLoading to true at the start of the function, so if the email changes the UI shows the loading indicator.